docs/3.0rc/deploy/control-runs/workers.mdx

```diff
@@ -0,0 +1,157 @@
+| `--work-queue`, `-q`                              | One or more work queue names for the worker to pull from. If not provided, the worker pulls from all work queues in the work pool.      |
+| `--type`, `-t`                                    | The type of worker to start. If not provided, the worker type is inferred from the work pool.                                          |
+| <span class="no-wrap">`--prefetch-seconds`</span> | The amount of time before a flow run's scheduled start time to begin submission. Default is the value of `PREFECT_WORKER_PREFETCH_SECONDS`. |
+| `--run-once`                                      | Only run worker polling once. By default, the worker runs forever.                                                                          |
+| `--limit`, `-l`                                   | The maximum number of flow runs to start simultaneously.                                                                                    |
+| `--with-healthcheck`                                   | Start a healthcheck server for the worker.                                                                                    |
+| `--install-policy`                                   | Install policy to use workers from Prefect integration packages.                                                                                    |
+
+You must start a worker within an environment to access or create the required infrastructure to execute flow runs. 
+The worker will deploy flow runs to the infrastructure corresponding to the worker type. For example, if you start a worker with 
+type `kubernetes`, the worker deploys flow runs to a Kubernetes cluster.
+
+Prefect must be installed in any environment (for example, virtual environment, Docker container) where you intend to run the worker or 
+execute a flow run.
+
+<Tip>
+**`PREFECT_API_URL` and `PREFECT_API_KEY`settings for workers**
+
+    `PREFECT_API_URL` must be set for the environment where your worker is running. You must also have a user or service account 
+    with the `Worker` role, which you can configure by setting the `PREFECT_API_KEY`.
+</Tip>
+
+### Worker status
+
+Workers have two statuses: `ONLINE` and `OFFLINE`. A worker is online if it sends regular heartbeat messages to the Prefect API. 
+If a worker misses three heartbeats, it is considered offline. By default, a worker is considered offline a maximum of 90 seconds 
+after it stopped sending heartbeats, but you can configure the threshold with the `PREFECT_WORKER_HEARTBEAT_SECONDS` setting.
+
+### Start a worker
+
+Use the `prefect worker start` CLI command to start a worker. You must pass at least the work pool name. 
+If the work pool does not exist, it will be created if the `--type` flag is used.
+<div class="terminal">
```

Comment:
```suggestion
```

Reply:
Let's remove the <div class=terminal>s and closing </div> that doesn't do anything any more.